.github/workflows/end2end.yaml

```diff
@@ -32,7 +32,7 @@ env:
   BACKBEAT_LCC_CRON_RULE: ""
   # e2e-env
   E2E_IMAGE_NAME: registry.scality.com/zenko/zenko-e2e
-  E2E_CTST_IMAGE_NAME: registry.scality.com/zenko/zenko-e2e-ctst
+  E2E_CTST_IMAGE_NAME: ghcr.io/scality/zenko/zenko-e2e-ctst
```

Comment:
not yet, keep pushing to all registry for now
(migration will be done separately)